.github/workflows/indent.yml

```diff
@@ -68,16 +68,3 @@ jobs:
       with:
         name: doxygen_documentation
         path: build/doc/doxygen
-
-  typos:
-    # check for typos
-
-    name: typos
-    runs-on: [ubuntu-24.04]
-
-    steps:
-    - uses: actions/checkout@v4
-    - uses: crate-ci/typos@v1.27.3
-      with:
-        files: doc examples include source tests
-        config: ./.typos.toml
```

Comment:
Yes they are opt-in locally and will only run on the changeset/git stage when calling `git commit` once activated.

But since the config defines some useful and sensible checks the CI job runs the checks on the full codebase just to be sure. 
So it is kind of the other way around. With the config and CI YAML we have a sensible CI check, 
which can easily be automated locally as well if you want to avoid getting "CI job failed" notifications minutes after updating your PR.

Reply:
OK. The point I wanted to make is that we *still* have to run these checks on the CI, even if they are available as pre-commit checks. Do I understand right that that they are still running?

Reply:
Ah got it.
Yes, a run of typos over the whole codebase is still performed as part of the CI.
But in `pre-commit` and no longer in `indent`.